lib/http/request/body.rb

```diff
@@ -32,14 +29,10 @@ def size
       #
       # @yieldparam [String]
       def each
-        return enum_for(__method__) unless block_given?
```

Comment:
Hm. If this works with issues on  JRuby, I think we should remove it then indeed :D